src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -323,15 +325,15 @@ public Builder addStdErrLineListener(ProcessOutputLineListener stdErrLineListene
      * The client listener of the process exit with code.
      */
     public Builder exitListener(ProcessExitListener exitListener) {
```

Comment:
potentially we would want this to be addExitListeners or whatever if we were exposing it as a list